packages/solana-programs-ts/tsconfig.cjs.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.base.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "outDir": "dist/cjs",
+    "target": "ES2016",
+    "module": "CommonJS",
+    "sourceMap": true,
+    "types": []
```

Comment:
This was interesting - not sure why, but leaving this out would freak out tsc as it would try to resolve all the implicit types at the root